cupertino/src/commonMain/kotlin/io/github/alexzhirkevich/cupertino/swipebox/SwipeBoxExpansionState.kt

```diff
@@ -0,0 +1,6 @@
+package io.github.alexzhirkevich.cupertino.swipebox
+
+data class SwipeBoxExpansionState(
```

Comment:
Public data classes inside libraries is antipattern. It can easily break binary compatibility

Reply:
That's good to know, I'll refactor this.